README.md

```diff
@@ -36,8 +38,8 @@ Examples:
 * SSH
     `$ cargo-temp anyhow=ssh://git@github.com/dtolnay/anyhow.git`
 
-This will add the repository on the default branch by default. You can choose
-another branch or a revision:
+This will add the repository on the last commit of the main branch by default. 
+You can choose another branch or a revision:
```

Comment:
It's not using the last commit, it's using the default branch.... I don't know why you changed that XD

Reply:
> Since we haven’t specified any other information, Cargo assumes that we intend to use the latest commit on the main branch to build our package.

Reply:
I mean yes it's technically correct. At the end of the day it is using the last commit of the default branch. But I just don't like the wording. 

But if you don't go into the details of Cargo and Git, what the user does by not specifying a branch nor a revision is to let cargo use its default behavior which is to use the default branch.

Now imagine that the repo changes of default branch (they switch from master to main). Then your Cargo.toml does NOT specify the commit it was before (last commit of master) but the new one (branch main).

I think what I don't like with the wording in cargo's doc is that it's mixing the lock file with the toml file. The user really intend to use the default branch of the repo and not a specific commit.

Reply:
Right, i see.